plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Cloud Tools for Eclipse App Engine New Project Generation
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.newproject;singleton:=true
+Bundle-Version: 0.1.0.qualifier
+Bundle-Vendor: Google, Inc.
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: org.eclipse.jface.viewers,
```

Comment:
These aren't right: `org.eclipse.jface.viewers` and `org.eclipse.jface.wizard` are packages, not bundles.